### PR TITLE
permissions-center: UX improvements.

### PR DESCRIPTION
**Pull request by sashaostrikov**
Including:
- Making provider state a badge + adding a tooltip if there is an error message;
- Make numbers in modal bold;
- Make alert animated.

Test plan:
Local sg run and manual tests.

### Demo

https://user-images.githubusercontent.com/94846361/225669076-528c78da-0004-4c4b-b6e3-54520f995920.mp4

## App preview:

- [Web](https://sg-web-ao-ui-pc-perms-dashboard-ux-fixes.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 1.64% (+46.25 kb) 🔺 | 0.00% (+0.65 kb) | -0.39% (-45.60 kb) 🔽 | -0.13% (-1) 🔽 |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ao/ui/pc/perms-dashboard-ux-fixes/compare-report.html;tab=live_object) for a full comparison between the commits 217fb1a and 9f04c08 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/site-admin/UserManagement/components/Table.tsx [hunk 1]
```diff
@@ -385,7 +385,12 @@ function Row<T>({
                     {render ? (
                         render(data, index)
                     ) : (
-                        <div className={styles.cell}>
+                        <div
+                            className={classNames(styles.cell, {
+                                [styles.alignLeft]: !align || align === 'left',
```
Comment (thenamankumar) [inline client/web/src/site-admin/UserManagement/components/Table.tsx:390]: I know JS is crazy, and `||` has higher precedence than `===`. 

But can you verify if `!align || align === 'left'` is same as `!align || (align === 'left')`

File: client/web/src/site-admin/permissions-center/PermissionsSyncJobsTable.tsx [hunk 2]
```diff
@@ -358,6 +363,31 @@ export const PermissionsSyncJobsTable: React.FunctionComponent<React.PropsWithCh
     )
 }
 
+// The Alert banner has a 1rem bottom margin
+const ONE_REM_IN_PX = convertREMToPX(1)
+const APPROX_BANNER_HEIGHT_PX = 40
+
+// TODO(sashaostrikov): extract as a separate wildcard component later
+const AnimatedAlert: React.FunctionComponent<React.PropsWithChildren<{ className?: string; visible: boolean }>> = ({
```
Comment (thenamankumar) [inline client/web/src/site-admin/permissions-center/PermissionsSyncJobsTable.tsx:371]: fancy shit

**Review by thenamankumar — APPROVED** [review]

File: client/web/src/site-admin/UserManagement/components/Table.tsx (hunk 1 above)
Comment (sashaostrikov) [reply client/web/src/site-admin/UserManagement/components/Table.tsx:390]: it is 👍🏻

**Pull request merged**